docs/conf/portland/2018/index.rst

```diff
@@ -1,6 +1,7 @@
 :template: 2018/na/index.html
 
-.. title:: Home - Write the Docs Portland 2018
+Home - Write the Docs Portland 2018
```

Comment:
Did this not blow up on rendering? There's a reason I didn't use this solution, but we _could_ easily make them all as orphans.

Reply:
As long as we don't use the title in the body of the index page template, should be fine right?

Reply:
I took a quick glance and it seemed ok